packages/manager/modules/anthos/src/dashboard/general-information/general-information.controller.js

```diff
@@ -0,0 +1,67 @@
+export default class {
+  /* @ngInject */
+  constructor($filter, $translate, CucCloudMessage, AnthosTenantsService) {
+    this.$filter = $filter;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
```

Comment:
As this module has no aim of being located in `public-cloud`, maybe we should prefer relying on `Alerter` from `@ovh-ux/ng-ovh-utils`. Even though it is deprecated it avoid importing cloud related components here (Also it is not declared as a dependency

Reply:
Changed